finalPrep/modZeroHero.js

```diff
@@ -1,84 +1,169 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "The Codenator";
+console.log(heroName);
+var specialAbility = "manifest real objects with code";
+console.log(specialAbility);
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
+var greeting = "Never fear, " + heroName + " is here!";
+console.log(greeting);
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var specialAbility = "manifest real objects with code";
+var catchPhrase = "You know, kinda like Green Lantern but without all the green glowing.";
+console.log(`I can ${specialAbility}. ${catchPhrase}`);
 
 // Declare two variables - power AND energy - set to integers
+var power = 15;
+var energy = 26;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
+var fullPower = power * 500;
+console.log(fullPower);
 //   fullEnergy should add 150 to your current energy
+var fullEnergy = energy + 150;
+console.log(fullEnergy);
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["Distracto", "The Procrastinator", "Debbie Downer"];
+var sidekicks  = ["Challenge Seeker", "Mr. Persistence", "Chronos"]
 // Print the first sidekick to your console
+console.log(sidekicks[0])
 
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2])
```

Comment:
Nice job with bracket notation. A next level to this task would be to come up with a way to print the last element of the array no matter how many elements are in it.